packages/@apphosting/adapter-angular/src/bin/build.spec.ts

```diff
@@ -35,14 +35,13 @@ describe("build commands", () => {
     const expectedFiles = {
       ".apphosting/dist/browser/browserfile": "",
       ".apphosting/dist/server/server.mjs": "",
-      ".apphosting/bundle.yaml": `headers: []
-redirects: []
-rewrites: []
+      ".apphosting/bundle.yaml": `
```

Comment:
Can we add another test that SSR_PORT is added for v17.3.2

Reply:
added